package.json

```diff
@@ -23,13 +23,13 @@
   "homepage": "https://github.com/silvermine/eslint-config-silvermine#readme",
   "dependencies": {
     "@silvermine/eslint-plugin-silvermine": "2.3.0-rc.1",
-    "@typescript-eslint/eslint-plugin": "2.16.0",
-    "@typescript-eslint/parser": "2.16.0",
     "typescript": "3.3.1"
   },
   "devDependencies": {
     "@commitlint/cli": "8.3.5",
     "@commitlint/travis-cli": "8.3.5",
+    "@typescript-eslint/eslint-plugin": "3.10.1",
+    "@typescript-eslint/parser": "3.10.1",
```

Comment:
Did you intend to move these to dev dependencies? That means any package using this package will need to declare these for themselves (which would really make them peer dependencies). I assume they should just remain in the regular deps list?

Reply:
Nope, I did not intend to do that 🤦